acceptance-tests/runner/runtime-plugins/runtime-deps/pom.xml

```diff
@@ -86,26 +86,26 @@
             <artifactId>structs</artifactId>
             <version>1.10</version>
         </dependency>
-        
+
         <dependency>
           <groupId>org.jenkins-ci.plugins</groupId>
           <artifactId>blueocean-executor-info</artifactId>
-          <version>0.1-beta-1</version>
+          <version>0.1-beta-3-SNAPSHOT</version>
```

Comment:
if confident, can publish this one and update it (or I guess do it after the merge)

Reply:
Wont this break ATH run? How will CI server find this dependency? I guess even if we want to use snapshot version, better to use timestamped one by releasing this as snapshot (mvn deploy)?

Reply:
ATH works fine, it's a deployed snapshot. It's temporary... can't release a non-snapshot until this goes beta. Fun cycles of dependencies!